### PR TITLE
Create TeamCity build to build docker images

### PR DESCRIPTION
**Pull request by scinos**
### Changes proposed in this Pull Request

Creates a new manual build in TeamCity. It will create a docker image using the base image (#45873), tag it as `calypso/app:build-<tc build number>` and `calypso/app:commit-<commit hash>`, and push it to the registry.

In a future PR I'll change `dserve` to pull those images from the registry.

### Testing instructions

* Functionality can't be tested until merged because TeamCity doesn't allow us to create new plans from a branch. As it is a manual build nothing will break, but the plan may require some tweaks once we can test it.

* Check the any TeamCity build for this PR and verify that it doesn't break when trying to apply the build settings (i.e. there are no syntax errors)

### CODE REVIEW

**Review by lynnbgriffith — APPROVED** [review]
👍 Makes sense.

File: .teamcity/settings.kts [hunk 1]
```diff
@@ -138,6 +140,56 @@ object BuildBaseImages : BuildType({
...
+				namesAndTags = """
+					registry.a8c.com/calypso/app:build-%build.number%
+					registry.a8c.com/calypso/app:commit-${WpCalypso.paramRefs.buildVcsNumber}
+				""".trimIndent()
+				commandArgs = "--pull --build-arg use_cache=true"
+			}
+			param("dockerImage.platform", "linux")
+		}
+		dockerCommand {
+			commandType = push {
```
Comment (saramarcondes) [inline .teamcity/settings.kts:168]: Does this need a `name`?

**Review by sarayourfriend — APPROVED** [review]

File: .teamcity/settings.kts (hunk 1 above)
Comment (scinos) [reply .teamcity/settings.kts:168]: Good catch!
I don't know, I exported this directly from TeamCity UI. Maybe names are optional and I forgot to add one?

Comment (scinos) [reply .teamcity/settings.kts:168]: Yep, they are optional:

image

**Pull request merged**